iac-common/src/main/java/org/sonar/iac/common/checks/PropertyUtils.java

```diff
@@ -48,6 +49,13 @@ public static Trilean has(@Nullable Tree tree, String key) {
     return Trilean.FALSE;
   }
 
+  public static boolean hasKeyWithValue(@Nullable Tree tree, String key, Collection<String> values) {
```

Comment:
IMO this method is too specific and the name is not self-explaining. Due to we already have a helper value, I could imagine a helper like `valueIs(@Nullable Tree tree, String key, Predicate<Tree>)` WDYT?

Reply:
You are right, I will change this method to have a less specific one.